packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -902,9 +887,23 @@ class ScrollableBaseNative extends Component {
 		this.isScrollAnimationTargetAccumulated = false;
 		this.scrolling = false;
 		this.forwardScrollEvent('onScrollStop');
+		this.startHidingThumb();
```

Comment:
If the timing of hiding is fine, it could be a good move. Please _feel_ it for sure. :)

Reply:
Do this change need to be applied to JS version, too?

Reply:
No, I do not think so. It's already doing it at `stop` https://github.com/enactjs/enact/blob/master/packages/ui/Scrollable/Scrollable.js#L936

Reply:
Okay!